src/core/a-scene.js

```diff
@@ -730,8 +730,8 @@ var AScene = module.exports = registerElement('a-scene', {
           stats('FPS').frame();
         }
         TWEEN.update(t);
-        this.behaviors.forEach(function (behavior) {
-          behavior.update();
+        this.behaviors.forEach(function (behaviorFn) {
```

Comment:
this should be:
`````javascript
This logic should be in the render loop. we should check:
````javascript
if (!this.paused) {
  this.behaviours.forEach(function(behaviour) {
     if (behaviour.el.paused) { return; }
     behavior.tick(t);
  });
}
````